lightning-tag/data-collection.md

```diff
@@ -1,42 +1,43 @@
 # Data Collection Guide
 
+!> Please remove all example values before copying on to your page.
+
 ## Passing Data on Tag Load
 
 Lightning Tag will automatically execute any data collection rules that you have configured in cooperation with your Lotame account representative. In addition, you can explicitly pass in data attributed to the user by supplying it in the `data` object of your tag input.
 
 ```javascript
-// Lotame Config
-var lotameTagInput = {
-  data: {
-    behaviorIds: [CSV_OF_BEHAVIOR_IDS],
-    behaviors: {
-      int: ['behaviorName', 'behaviorName2'],
-      act: ['behaviorName']
-    },
-    ruleBuilder: {
-      key1: ['value 1a', 'value 1b']
-    },
-    thirdParty: {
-      namespace: 'NAMESPACE',
-      value: 'TPID_VALUE'
+! function() {
+  var lotameClientId = '<lotameClientId>';
+
+  // Lotame Config
+  var lotameTagInput = {
+    data: {
+      behaviorIds: [1,2,3],
+      behaviors: {
+        int: ['behaviorName', 'behaviorName2'],
+        act: ['behaviorName']
+      },
+      ruleBuilder: {
+        key1: ['value 1a', 'value 1b']
+      },
+      thirdParty: {
+        namespace: 'NAMESPACE',
+        value: 'TPID_VALUE'
+      },
+      sha256email: 'SHA256_VALUE'
     },
-    sha256email: 'SHA256_VALUE'
-  },
-  config: {
-    clientId: <lotameClientId>,
-    audienceLocalStorage: true, // written to 'lotame_<lotameClientId>_auds' key
```

Comment:
you dropped these two lines

on purpose?  onProfileReady is important isn't it?  is this just now somewhere else?

Reply:
It was not the main point of the example being presented so I thought it excessive

Reply:
alright.  that part is for targeting and this example is collection?

Reply:
exactly. I'm trying to limit number of places that need to be changed when we make updates